server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds.Contracts/Arcadia.Assistant.AppCenterBuilds.Contracts.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
only actual services should target netcore. Libraries should be netstandard.

Reply:
Done